stdeb.cfg

```diff
@@ -12,7 +12,7 @@ Conflicts: python3-rosdistro
 Conflicts3: python-rosdistro
 Copyright-File: LICENSE.txt
 Suite: bionic cosmic disco eoan buster
-Suite3: bionic cosmic disco eoan focal jammy buster bullseye
+Suite3: bionic cosmic disco eoan focal jammy buster bullseye bookworm
```

Comment:
```suggestion
Suite3: focal jammy noble bookworm trixie
```

Reply:
+1 to Scott's suggested changes. We've dropped our support for interim distros on the ROS build farm.